internal/multigitter/review.go

```diff
@@ -0,0 +1,253 @@
+package multigitter
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/lindell/multi-gitter/internal/scm"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// Reviewer reviews pull requests in an organization
+type Reviewer struct {
+	VersionController VersionController
+	FeatureBranch     string
+	Comment           string
+	All               bool
+	Batch             string
+	Pager             string
+	DisablePaging     bool
+	IncludeApproved   bool
+}
+
+const (
+	header        = "# *********************************************************"
+	reviewOptions = "[a/r/c/n/d/h (help)]"
+	batchApprove  = "approve"
+	batchReject   = "reject"
+	batchComment  = "comment"
+)
+
+type approvdPr struct {
+	scm.PullRequest
+	approved bool
+}
+
+// Reviewer reviews pull requests in an organization
+func (s Reviewer) Review(ctx context.Context) error {
+	prs, err := s.VersionController.GetPullRequests(ctx, s.FeatureBranch)
+	if err != nil {
+		return err
+	}
+
+	var approvedPrs []approvdPr
+
+	for _, pr := range prs {
+		approved, err := s.VersionController.IsPullRequestApprovedByMe(ctx, pr)
+		if err != nil {
+			log.Errorf("Failed to retrieve pull request reviews: %s", err.Error())
+			continue
+		}
+
+		if approved && !s.IncludeApproved {
+			continue
+		}
+
+		approvedPrs = append(approvedPrs, approvdPr{
+			PullRequest: pr,
+			approved:    approved,
+		})
+	}
+
+	if len(approvedPrs) == 0 {
+		fmt.Println("All pull requests are approved by you. You can still view and comment them by using review --include-approved")
```

Comment:
Should probably have a text for `len(prs) == 0`, so that you don't get that you have all approved PRs when none exist, or you misspelled the branch-name 😉

Reply:
Yep :) 👍🏻